src/contentmanagermodel.cpp

```diff
@@ -123,10 +123,7 @@ void ContentManagerModel::setBooksData(const BookInfoList& data, const Downloads
         const auto rowNode = createNode(bookItem);
 
         // Restore download state during model updates (filtering, etc)
-        const auto downloadIter = downloads.constFind(rowNode->getBookId());
-        if ( downloadIter != downloads.constEnd() ) {
-            rowNode->setDownloadState(downloadIter.value());
-        }
+        rowNode->setDownloadState(downloads.value(rowNode->getBookId()));
```

Comment:
I am using [`QMap::value(const Key &key, const T &defaultValue = T())`](https://doc.qt.io/qt-5/qmap.html#value) (rather than [`QMap::operator[](const Key &key)`](https://doc.qt.io/qt-5/qmap.html#operator-5b-5d)) as a more concise way of performing the absence-of-the-key test.